e2e/libs/backupstore/__init__.py

```diff
@@ -0,0 +1,2 @@
+from backupstore.nfs import Nfs
+from backupstore.minio import Minio
```

Comment:
GitHub is showing a red marker (No newline at end of file) for some files in the PR. This is probably not harmful, but it can be rather bothersome. Suggest adding a new line at the end of those files to resolve it.

Reply:
Refined. Thanks!